### PR TITLE
Add synonymous forms for reverse navigation on device and service to …

### PR DESCRIPTION
**Pull request by otaviojacobi**
…device service environment variable

Change-type: patch

### CODE REVIEW

File: src/balena.sbvr [hunk 1]
```diff
@@ -814,8 +816,14 @@ Fact type: device service environment variable has value
 	Necessity: each device service environment variable has exactly one value.
 Fact type: device service environment variable has service
 	Necessity: each device service environment variable has exactly one service.
+	-- This is the default name that will be given to the synonymous form
+	-- once the device service environment variable origin fact type changes
+	Synonymous form: service has device service environment variable
```
Comment (thgreasi) [inline src/balena.sbvr:821]: NIT: Usually we place the `Synonymous form` right below the `Fact type`, and Necessities come right after

**Review by thgreasi — APPROVED** [review]
I would expect to also see the changes that use this in the same PR  (since these are not exposed atm on a stable versioned endpoint) which also proves that the changes work, but I'm also fine w/ this PR as is :+1:

File: src/balena.sbvr (hunk 1 above)
Comment (otaviojacobi) [reply src/balena.sbvr:820]: Done, thanks

**Comment by thgreasi** [comment]
go for it

**Comment by thgreasi** [comment]
mmmmmmmerge it

**Pull request merged**